### PR TITLE
Accordion in Labs

### PR DESCRIPTION
**Pull request by jordankoschei-okta**

### CODE REVIEW

File: packages/odyssey-react-mui/src/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   */
+  children: ReactNode;
+  /**
+   * The label text for the AccordionSummary
+   */
+  label: string;
+  /**
+   * If true, the Accordion item won't have a shadow.
+   */
+  hideShadow?: boolean;
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/Accordion.tsx:36]: `hasShadow` (you gotta swap the boolean though)

File: packages/odyssey-react-mui/src/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  onChange,
+}: AccordionProps) => {
+  return (
+    <MuiAccordion
+      defaultExpanded={isDefaultExpanded}
+      disabled={isDisabled}
+      disableGutters
+      expanded={isExpanded}
+      onChange={onChange}
+      className={hideShadow ? `noShadow` : undefined}
```
Comment (jordankoschei-okta) [inline packages/odyssey-react-mui/src/Accordion.tsx:71]: rename to `hasShadow`

File: packages/odyssey-react-mui/src/Accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+   */
+  isDefaultExpanded?: boolean;
+  /**
+   * Whether the item is disabled
+   */
+  isDisabled?: boolean;
+  /**
+   * Whether the item is expanded
+   */
+  isExpanded?: boolean;
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/Accordion.tsx:48]: This type needs to be either-or for using the default or using `isExpanded`

File: packages/odyssey-react-mui/src/theme/components.tsx [hunk 4]
```diff
@@ -59,6 +59,98 @@ export const components = ({
...
+
+            "& .MuiAccordionSummary-content": {
+              color: odysseyTokens.TypographyColorDisabled,
+            },
+          },
+
+          "&::before": {
+            backgroundColor: odysseyTokens.BorderColorDisplay,
+            opacity: "1 !important",
+          },
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/theme/components.tsx:88]: Can we remove these `!important` declarations?

We can hack it like `&::before::before` though I think.

File: packages/odyssey-react-mui/src/theme/components.tsx [hunk 5]
```diff
@@ -59,6 +59,98 @@ export const components = ({
...
+        root: () => ({
+          backgroundColor: odysseyTokens.HueNeutralWhite,
+          borderWidth: 1,
+          borderInlineStyle: "solid",
+          borderColor: odysseyTokens.HueNeutral100,
+          boxShadow: odysseyTokens.DepthMenu,
+
+          "&.noShadow": {
+            boxShadow: "none !important",
+          },
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/theme/components.tsx:73]: Can we remove this `!important`?

File: packages/odyssey-react-mui/src/theme/components.tsx [hunk 6]
```diff
@@ -59,6 +59,98 @@ export const components = ({
...
+            borderBottomRightRadius: 0,
+          },
+
+          "&:hover, &:focus": {
+            backgroundColor: odysseyTokens.HueNeutral50,
+          },
+
+          "&:focus-visible": {
+            outlineColor: odysseyTokens.PalettePrimaryMain,
+            outlineWidth: 2,
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-react-mui/src/theme/components.tsx:131]: Do we have a token value for `outlineWidth`?

**Review by KevinGhadyani-Okta — APPROVED** [review]
Just a few small comments. If we make tickets for those, I'm fine.

**Pull request merged**